includes/admin/settings/class-wc-settings-accounts.php

```diff
@@ -96,6 +96,14 @@ public function get_settings() {
 				'id'            => 'woocommerce_registration_generate_password',
 				'default'       => 'yes',
 				'type'          => 'checkbox',
+				'checkboxgroup' => '',
+				'autoload'      => false,
+			),
+			array(
+				'desc'          => __( 'When creating an account, automatically generate an account password overwrite', 'woocommerce' ),
+				'id'            => 'woocommerce_registration_generate_password_overwrite',
+				'default'       => 'yes',
+				'type'          => 'checkbox',
 				'checkboxgroup' => 'end',
 				'autoload'      => false,
```

Comment:
A few comments:

1. If possible, this option should only be clickable in the admin after you have enabled `woocommerce_registration_generate_password` as it doesn't make sense to be active on its own.
2. For claritys sake, I would consider using the following namings:
- id: woocommerce_registration_generate_password_allow_overwrite
- desc: When creating an account, allow customers to overwrite automatic password generation by optionally supplying a password